core/src/main/scala/stryker4s/run/ThresholdChecker.scala

```diff
@@ -0,0 +1,23 @@
+package stryker4s.run
+
+import grizzled.slf4j.Logging
+import stryker4s.config.Config
+import stryker4s.model.MutantRunResults
+
+class ThresholdChecker extends Logging {
+
+  def determineExitCode(runResults: MutantRunResults)(implicit config: Config): Int = {
+    config.thresholds match {
+      case None =>
```

Comment:
Based on the previous comment we can also then remove this. Am not a fan of this logging statement, to begin with. This is because of the log statements kinda implying you need to specify this.

Reply:
[I actually copied this logging from stryker](https://github.com/stryker-mutator/stryker/blob/master/packages/stryker/src/ScoreResultCalculator.ts#L30).

I could rewrite it to something that doesn't imply that. But since this is logged as DEBUG anyway I thought it could be useful. 
(Could rewrite it to `Set 'threshold.break' if you want to change this behavior`.)